From: YOKOTA Hiroshi <yokota.hgml@gmail.com>
Date: Sun, 19 Feb 2023 23:14:50 +0900
Subject: Don't download from CA certificates

Use Debian files at /etc/ssl/certs/ca-certificates.crt
---
 setup/resources.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/setup/resources.py b/setup/resources.py
index 60168ee..457d1fe 100644
--- a/setup/resources.py
+++ b/setup/resources.py
@@ -136,6 +136,8 @@ class CACerts(Command):  # {{{
     CA_PATH = os.path.join(Command.RESOURCES, 'mozilla-ca-certs.pem')
 
     def run(self, opts):
+        return
+
         try:
             with open(self.CA_PATH, 'rb') as f:
                 raw = f.read()
